lib/dialogs/promo/connected_wallet_promo_dialog.dart

```diff
@@ -0,0 +1,134 @@
+                            color: Colors.white,
+                          ),
+                          onPressed: () => Navigator.pop(context),
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: [
+                    Container(
+                      width: 55,
+                      height: 55,
+                      decoration: const BoxDecoration(
+                        shape: BoxShape.circle,
+                        image: DecorationImage(
+                          image: AssetImage(
+                            'assets/images/x.jpg',
+                          ),
+                        ),
+                      ),
+                    )
+                  ],
+                ),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    const Text(
+                      'Did you learn about AX from an existing AX User?',
+                      style: TextStyle(
+                        fontSize: 16,
+                        color: Colors.white,
+                      ),
+                      textAlign: TextAlign.center,
+                    ),
+                    const SizedBox(height: 15),
+                    const Text(
+                      ' Use the following link to join our discord'
+                      ' and shout them out and both of you will receive'
+                      ' 1000 AX!',
+                      style: TextStyle(
+                        fontSize: 16,
+                        color: Colors.white,
+                      ),
+                      textAlign: TextAlign.center,
+                    ),
+                    const SizedBox(height: 10),
+                    Container(
+                      width: 275,
+                      height: 50,
+                      decoration: BoxDecoration(
+                        color: Colors.transparent,
+                        borderRadius: BorderRadius.circular(100),
+                      ),
+                      child: TextButton(
+                        onPressed: () {
+                          launchDiscordLink();
+                          Navigator.pop(context);
+                        },
+                        child: const Text(
+                          'Claim Your AX!',
+                          style: TextStyle(
+                            fontSize: 19,
+                            color: Colors.yellow,
+                          ),
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  void launchDiscordLink() {
+    const url = 'https://discord.com/channels/761052062184112148/1016454317403807814 ';
```

Comment:
Was this provided by @robkizzle so that we can track it?

Reply:
@robkizzle gave me the right link so I will update that